src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -555,11 +555,11 @@ else if( player.capabilities.isCreativeMode )
 			return;
 		}
 
-		if( !this.disableShiftClick && isShiftKeyDown() )
+		if( !this.disableShiftClick && isShiftKeyDown() && mouseButton == 0)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)